hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/ChunkManagerDummyImpl.java

```diff
@@ -42,8 +40,6 @@
  * Chunks are not written to disk, Reads are returned with zero-filled buffers
  */
 public class ChunkManagerDummyImpl extends ChunkManagerImpl {
-  private static final Logger LOG = LoggerFactory.getLogger(
-      ChunkManagerDummyImpl.class);
```

Comment:
Sorry, I didn't notice this one earlier: we should keep this, since `writeData` uses it.  (If removed it will use the one inherited from `ChunkManagerImpl`, resulting in wrong class name in log message.)

Reply:
Yes, thanks. And the LOG should be private in ChunkManagerImpl to avoid such mistakes.